python/hsfs/core/feature_monitoring_config_api.py

```diff
@@ -0,0 +1,278 @@
+#
+#   Copyright 2024 Hopsworks AB
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+from typing import List, Optional
+from hsfs import client
+from hsfs.core import feature_monitoring_config as fmc
+from hsfs.core.job import Job
+
+
+class FeatureMonitoringConfigApi:
+    """Feature Monitoring Configuration endpoints for the Feature Group resource.
+    :param feature_store_id: id of the respective Feature Store
+    :type feature_store_id: int
+    :param feature_group_id: id of the feature group, if monitoring a feature group
+    :type feature_group_id: int, optional
+    :param feature_view_name: name of the feature view, if monitoring a feature view
+    :type feature_view_name: str, optional
+    :param feature_view_version: version of the feature view, if monitoring a feature view
+    :type feature_view_version: int, optional
+    """
+
+    def __init__(
+        self,
+        feature_store_id: int,
+        feature_group_id: Optional[int] = None,
+        feature_view_name: Optional[str] = None,
+        feature_view_version: Optional[int] = None,
```

Comment:
can you add **kwargs to all the init methods?

Reply:
done